runtime/calamari/src/weights/manta_collator_selection.rs

```diff
@@ -74,6 +76,13 @@ impl<T: frame_system::Config> manta_collator_selection::WeightInfo for Substrate
 		(10_523_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
+
+	fn set_eviction_percentile() -> Weight{
+		(0 as Weight)
```

Comment:
Yeah, is was trying to figure out how to run benchmark locally, but my machine is not up to spec.
Once CI succeeds i hope I can just use the CI benchmark artifact

Reply:
Cool

Reply:
@Garandor please search for `Search for Generating weight files with CI workflows` in the new benchmarking doc in Notion, to see how to use the CI for benchmarking.

Reply:
Added CI-computed weights.
Most extrinisics are cheaper at the tradeoff that the note-author hook and new_session access more storage items.
new_session might be cheaper for large N of collators because the map is only read once ( weights file shows twice for some reason don't see ) instead of the old solution that had N reads.